src/main/java/de/oppermann/bastian/safetrade/listener/InventoryDragListener.java

```diff
@@ -27,9 +26,7 @@
      */
     @EventHandler
     public void onInventoryDrag(final InventoryDragEvent event) {
-        if (event.getWhoClicked().getOpenInventory() == null) {
-            return;
-        }
+        event.getWhoClicked().getOpenInventory();
```

Comment:
Why did you remove this check? Just calling the getter seems pretty useless?

Reply:
oh, missed that, i intendes to remove the entire check as the getter is marked as @NonNull